server/protocols/jmap-integration-testing/memory-jmap-integration-testing/src/test/java/org/apache/james/jmap/memory/FileMailRepositoryDeletedMessagesVaultTest.java

```diff
@@ -1,64 +0,0 @@
-/****************************************************************
- * Licensed to the Apache Software Foundation (ASF) under one   *
- * or more contributor license agreements.  See the NOTICE file *
- * distributed with this work for additional information        *
- * regarding copyright ownership.  The ASF licenses this file   *
- * to you under the Apache License, Version 2.0 (the            *
- * "License"); you may not use this file except in compliance   *
- * with the License.  You may obtain a copy of the License at   *
- *                                                              *
- *   http://www.apache.org/licenses/LICENSE-2.0                 *
- *                                                              *
- * Unless required by applicable law or agreed to in writing,   *
- * software distributed under the License is distributed on an  *
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
- * KIND, either express or implied.  See the License for the    *
- * specific language governing permissions and limitations      *
- * under the License.                                           *
- ****************************************************************/
-
-package org.apache.james.jmap.memory;
-
-import java.io.IOException;
-import java.time.Clock;
-
-import org.apache.james.GuiceJamesServer;
-import org.apache.james.MemoryJmapTestRule;
-import org.apache.james.filesystem.api.FileSystem;
-import org.apache.james.jmap.methods.integration.DeletedMessagesVaultTest;
-import org.apache.james.mailrepository.api.MailRepositoryUrl;
-import org.apache.james.modules.vault.TestDeleteMessageVaultPreDeletionHookModule;
-import org.apache.james.vault.MailRepositoryDeletedMessageVault;
-import org.apache.james.webadmin.WebAdminConfiguration;
-import org.junit.Ignore;
-import org.junit.Rule;
-import org.junit.Test;
-
-public class FileMailRepositoryDeletedMessagesVaultTest extends DeletedMessagesVaultTest {
```

Comment:
Why did you delete this file?

Reply:
this file uses MailRepositoryDeletedMessageVault which relies on the FileMailRepository, and it's inside memory product. Since we switch to use BlobStoreDeletedMessageVault on memory for memory product. This test doesn't add value as I think.

Reply:
We still have `MailRepositoryDeletedMessageVault` thus the file composition still needs to be tested. Let's not remove things too fast.

Reply:
then I have to revert the MemoryMailRepositoryDeletedMessagesVaultTest also, I had deleted it